tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -48,6 +48,7 @@ internal class TracerManagerFactory
         internal TracerManager CreateTracerManager(ImmutableTracerSettings settings, TracerManager previous)
         {
             // TODO: If relevant settings have not changed, continue using existing statsd/agent writer/runtime metrics etc
+            // If reusing the runtime metrics/statsd, need to propagate the new value of DD_TAGS from dynamic config
```

Comment:
This is an amendment to the TODO above right, not a new TODO?

Reply:
Correct